lang/po/new/cs.po

```diff
@@ -78,7 +76,7 @@ msgstr "Okoli"
 # MSG_AUTO c=6
 #: messages.c:160
 msgid "Auto"
-msgstr ""
+msgstr "Auto"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.